index.html

```diff
@@ -0,0 +1,200 @@
+
+        const personal1 = [{ id: 50, name: "Fancy FitzGilbert", birthday: "09-03-1999", employeedCode: 08 - 8870128, adress: "3 Blue Bill Park Crossing", email: "ffitzgilbert1d@hugedomains.com", phone: "238-773-0567" }];
+        $(document).ready(function () {
+            renderTable();
+        });
+
+        const pageSize = 5;
+        let curPage = 1;
+
+        $("#btnPrevious").on("click", clickPre);
+        $("#btnNext").on("click", clickNext);
+
+        function clickPre() {
+            if (curPage > 1)
+                curPage--;
+            renderTable();
+        }
+
+        function clickNext() {
+            if (curPage * pageSize < personal.length)
+                curPage++;
+            renderTable();
+        }
+
+        function loadData() {
+            var i;
+            var html = '';
+            for (i = 0; i < personal.length; i++) {
+                let tRow = document.createElement('tr');
+
+                let tId = document.createElement('td');
+                tId.innerText = personal[i].id;
+
+                let tName = document.createElement('td');
+                tName.innerText = personal[i].name;
+
+                let tBirthday = document.createElement('td');
+                tBirthday.innerText = personal[i].birthday;
+
+                let tcode = document.createElement('td');
+                tcode.innerText = personal[i].employeedCode;
+
+                let tadress = document.createElement('td');
+                tadress.innerText = personal[i].adress;
+
+                let tEmail = document.createElement('td');
+                tEmail.innerText = personal[i].email;
+
+                let tPhone = document.createElement('td');
+                tPhone.innerText = personal[i].phone;
+
+                tRow.appendChild(tId);
+                tRow.appendChild(tName);
+                tRow.appendChild(tBirthday);
+                tRow.appendChild(tcode);
+                tRow.appendChild(tadress);
+                tRow.appendChild(tEmail);
+                tRow.appendChild(tPhone);
+
+                document.getElementById('myBody').appendChild(tRow);
+            }
+
+
+
+        }
+
+        function renderTable() {
+            let result = '';
+
+            personal.filter((row, index) => {
+                let start = (curPage - 1) * pageSize;
+                let end = curPage * pageSize;
+                if (index >= start && index < end)
+                    return true;
+                if (end === (personal.length) && start === (personal.length - pageSize))
+                    $("#btnNext").hide();
+                else {
+                    $("#btnNext").show();
+                }
+                if (end === pageSize && start === 0)
+                    $("#btnPrevious").hide();
+                else {
+                    $("#btnPrevious").show();
+                }
```

Comment:
theo a đoạn này đưa được ra ngoài vòng for (filter) thì hay hơn, mỗi lần next hoặc previous thì chỉ cần check 1 lần thôi
còn như thế kia với mỗi lần next hoặc previous phải check 5 lần